spec/react_on_rails/support/shared_examples/base_generator_examples.rb

```diff
@@ -95,7 +95,7 @@
       Rails.application.config.assets.paths << Rails.root.join("client", "assets", "stylesheets")
       Rails.application.config.assets.paths << Rails.root.join("client", "assets", "images")
       Rails.application.config.assets.paths << Rails.root.join("client", "assets", "fonts")
-      Rails.application.config.assets.precompile += %w( generated/server-bundle.js )
+      Rails.application.config.assets.precompile += %w( generated/server-bundle.js react_on_rails )
```

Comment:
This seems odd, as the rails installation should have the Gem's JS file. We're just probably looking in the wrong place.